webapp/src/components/Question.vue

```diff
@@ -93,32 +110,34 @@ export default {
     showSolution: false,
     theresNoSolution: false,
     solutionNotFound: "Não identificamos nenhum problema!",
+    feedbackNps:
+      "Gostaria de nos ajudar a melhorar essa ferramenta? envie seu feedback!",
     idStage: 1,
     isTypewriterRunning: false,
     callBack: () => {},
     disableButtonNotUnderstand: false,
-    typewritingQuestion: "",
-
+    typewritingQuestion: ""
   }),
 
   created() {
-      StageService.getStageById(this.idStage).then(response => {
-      let stage = response.data;
-      this.questionList = stage.questions;
-      this.nextQuestion();
-    })
-    .catch((error) => {
-    });
+    StageService.getStageById(this.idStage)
+      .then(response => {
+        let stage = response.data;
```

Comment:
Qual a motivação por trás disso? Pq não fazemos direto `this.questionList = response.data.questions`?